lms/envs/acceptance.py

```diff
@@ -195,3 +195,7 @@ def seed():
 ############################### PIPELINE #######################################
 
 PIPELINE_ENABLED = False
+
+# We want to make sure that any new migrations are run
+# see https://groups.google.com/forum/#!msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ
```

Comment:
As the thread discusses, how will this impact our test times? Obviously we need to be testing against a fully-migrated test DB, so I think this change is functionally correct.